edx_django_utils/user/management/tests/test_manage_user.py

```diff
@@ -0,0 +1,184 @@
+"""
+Unit tests for user_management management commands.
+"""
+
+
+import itertools
+
+import ddt
+import pytest
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth.models import Group, User  # lint-amnesty, pylint: disable=imported-auth-user
```

Comment:
Instead of importing `User` directly, please import `get_user_model` from `django.contrib.auth`.

Then, assign `User = get_user_model()`.

That will allow this command to work on services that define a custom `User` model.

Reply:
Done